systemtest/src/test/resources/upgrade/StrimziUpgradeST.yaml

```diff
@@ -24,11 +25,12 @@
 #  featureGatesAfter: String - FG added to `STRIMZI_FEATURE_GATES` environment variable, on upgrade of CO
 #  --- Structure ---
 
-- fromVersion: 0.33.2
-  fromExamples: strimzi-0.33.2
+- fromVersion: 0.32.0
```

Comment:
This is non-sense, we don't want to have `0.32.0` here as we are always testing upgrade from latest release to "latest" -> "main" branch.
And same for all other fields you changed in this file